specs/binding-proxy.spec/src/main/resources/META-INF/zilla/proxy.idl

```diff
@@ -127,4 +127,36 @@ scope proxy
             ProxyInfo[] infos;
         }
     }
+
+    scope event
+    {
+        enum ProxyEventType (uint8)
+        {
+            REMOTE_ACCESS (1),
+            TLS (2)
```

Comment:
Let's example on this TLS event type to make sure we are clear on what needs to be captured.